AsyncDisplayKit/ASTextNode.mm

```diff
@@ -1016,9 +998,15 @@ - (void)setShadowRadius:(CGFloat)shadowRadius
   }
 }
 
+//only safe to call on main thread
```

Comment:
Could you add just a couple words to explain what about this method is not thread safe?  Shouldn't require a multi line explanation, but I am left wondering if it is the shadow padding method or the renderer call.